Solves #3 : Adds Some description and screenshot to readme (#21)
kyzn/PRC: README.md

# PullRequest.Club

> [Pull Request Club](https://pullrequest.club) is a web application where users can sign up with their Github accounts to receive monthly Github issues to work. Its purpose is to connect

Reviewer: ```suggestion
[Pull Request Club](https://pullrequest.club) is an application where users can sign up with their GitHub accounts. It lets users receive assignments and/or assignees.

PRC aims to connect people who are looking for an open source repository to work on with people who are looking for new contributors for their projects.
```

> - people who are looking for an open source project to work on

Reviewer: ```suggestion
```

> - with the people who want new contributors for their project

Reviewer: ```suggestion
```

> Check Pull Request Club's [help](https://pullrequest.club/help) section for more information.

Reviewer: ```suggestion
See [help](https://pullrequest.club/help) for more information.
```

> ### Installation & Deployment

Reviewer: ```suggestion
### Development
```

> - [Installation Instructions](https://github.com/kyzn/PRC/wiki/Development-Instructions)

Reviewer: ```suggestion
- [Development Instructions](https://github.com/kyzn/PRC/wiki/Development-Instructions)
```

- [Deployment Instructions](https://github.com/kyzn/PRC/wiki/Deployment-Instructions)

See [wiki](https://github.com/kyzn/PRC/wiki) for further instructions.

> ### Screenshot

Reviewer: ```suggestion
```
